### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jeffallen**
Fix broken link.

### CODE REVIEW

**Comment by shreevatsa** [comment]
I'm just a random person who got notified of this (maybe because I starred or clicked "Watch" on this repo) but it may be preferable to use the current URL (https://onlinelibrary.wiley.com/doi/abs/10.1002/spe.4380111102) or even better the permanent URL (https://doi.org/10.1002/spe.4380111102) rather than the Wayback Machine's historical URL.

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ## TeX line breaking algorithm in JavaScript
 
-This is an implementation of the [Knuth and Plass line breaking algorithm](http://www3.interscience.wiley.com/journal/113445055/abstract) using JavaScript. The goal of this project is to optimally set justified text in the browser, and ultimately provide a library for various line breaking algorithms in JavaScript.
+This is an implementation of the [Knuth and Plass line breaking algorithm](https://web.archive.org/web/20110930221135/http://onlinelibrary.wiley.com/doi/10.1002/spe.4380111102/abstract) using JavaScript. The goal of this project is to optimally set justified text in the browser, and ultimately provide a library for various line breaking algorithms in JavaScript.
```
Comment (jeffallen) [inline README.md]:
```suggestion
This is an implementation of the [Knuth and Plass line breaking algorithm](https://doi.org/10.1002/spe.4380111102) using JavaScript. The goal of this project is to optimally set justified text in the browser, and ultimately provide a library for various line breaking algorithms in JavaScript.
```

File: README.md [hunk 2]
```diff
@@ -77,6 +77,6 @@ The following are some extensions to the algorithm discussed in the original pap
 These are the resources I found most useful while implementing the line breaking algorithm.
 
 * [Digital Typography, Donald E. Knuth](http://www.amazon.com/Digital-Typography-Center-Language-Information/dp/1575860104/)
-* [Breaking paragraphs into lines, Donald E. Knuth, Michael F. Plass](http://www3.interscience.wiley.com/journal/113445055/abstract)
+* [Breaking paragraphs into lines, Donald E. Knuth, Michael F. Plass](https://web.archive.org/web/20110930221135/http://onlinelibrary.wiley.com/doi/10.1002/spe.4380111102/abstract)
```
Comment (jeffallen) [inline README.md]:
```suggestion
* [Breaking paragraphs into lines, Donald E. Knuth, Michael F. Plass](https://doi.org/10.1002/spe.4380111102)
```

**Comment by jeffallen** [comment]
The only problem with the "correct" URL is that Wiley now wants to charge for access to the article, whereas in the past they allowed open access to it.

**Comment by cben** [comment]
Maybe also link to Michael Plass's thesis https://www.tug.org/docs/plass/plass-thesis.pdf

**Comment by justinpenner** [comment]
> The only problem with the "correct" URL is that Wiley now wants to charge for access to the article, whereas in the past they allowed open access to it.

Until a non-paywalled link is found, the Wiley link is more useful than a 403 Forbidden error.

**Comment by shreevatsa** [comment]
> The only problem with the "correct" URL is that Wiley now wants to charge for access to the article, whereas in the past they allowed open access to it.

For what it's worth I can't access the full PDF via [the Wayback Machine URL](https://web.archive.org/web/20110930221135/http://onlinelibrary.wiley.com/doi/10.1002/spe.4380111102/abstract) you had earlier either, and as far as I can tell, I don't think they ever allowed open access to this paper. However one can easily find copies online by searching for the appropriate keywords, plus with the DOI there are… ways.

**Pull request closed without merging**